modules/planning/tasks/deciders/path_reuse_decider/path_reuse_decider.cc

```diff
@@ -334,5 +338,41 @@ bool PathReuseDecider::IsCollisionFree(
   return true;
 }
 
+bool PathReuseDecider::TrimHistoryPath(
+    Frame* const frame, ReferenceLineInfo* const reference_line_info) {
+  const auto& history_frame = FrameHistory::Instance()->Latest();
+  if (!history_frame) return false;
+  const DiscretizedPath& history_path =
+      history_frame->current_frame_planned_path();
+  DiscretizedPath trimmed_path;
+  // current vehicle status
+  common::math::Vec2d adc_position = {
+      common::VehicleStateProvider::Instance()->x(),
+      common::VehicleStateProvider::Instance()->y()};
+  common::SLPoint adc_position_sl;
+  const auto& reference_line = reference_line_info->reference_line();
+  reference_line.XYToSL(adc_position, &adc_position_sl);
+  double path_start_s = 0;
```

Comment:
Would 0.0 be a better format for double initialization here?

Reply:
Updated.

Reply:
Thanks.